numpy/lib/function_base.py

```diff
@@ -3854,7 +3854,7 @@ def _median(a, axis=None, out=None, overwrite_input=False):
 
 
 def _percentile_dispatcher(a, q, axis=None, out=None, overwrite_input=None,
-                           interpolation=None, keepdims=None):
+                           method=None, keepdims=None, *, interpolation=None):
```

Comment:
Hmm.  The change in order is probably safe. We should make the options keyword only at some point.

Reply:
NVM, this is private, so no problem.

Reply:
This is the dispatch, so it is the same as the public signature.  This is something `__array_function__` implementers will have to catch up with.

I guess one annoyance is that it is hard to use avoid the warning if you are writing for both old and new NumPy versions.  That would be an argument to delay the `DeprecationWarning` for a bit...

Reply:
Let's see how the first release goes.